app/src/main/java/com/sopt/now/main/MyPageFragment.kt

```diff
@@ -40,18 +41,26 @@ class MyPageFragment(
         _binding = null
     }
 
-
     private fun initTextViews(user: User) {
         with(binding){
             myPageTvNickname.text = user.nickName
-            myPageTvIntro.text = getString(R.string.mypage_tv_mbti, user.mbti)
+            myPageTvIntro.text = getString(R.string.mypage_tv_phone_num, user.phoneNum)
             myPageTvId.text = user.id
-            myPageTvPw.text = user.pw
         }
     }
 
     private fun initButton() {
-        binding.myPageBtnLogout.setOnClickListener(onClickLogoutButton)
+        with(activity as MainActivity){
+            binding.myPageBtnChangePw.setOnClickListener {
+                val intent = Intent(this, PasswordActivity::class.java)
+                startActivity(intent)
```

Comment:
```suggestion
                Intent(this, PasswordActivity::class.java).apply{
                  startActivity(this)
                }
```
처럼 스코프 함수를 활용하면 불필요한 객체 생성을 줄일 수 있어요!

Reply:
`apply`의 경우 방금 생성한 객체를 다른 메서드나 변수 초기화에 사용하기 위해 사용하는 스코프인가 보네요. 더 자세히 알아보고 적용해보겠습니다!!